apps/crn-server/test/fixtures/research-output.fixtures.ts

```diff
@@ -87,6 +87,7 @@ export const getSquidexGraphqlResearchOutput = (): NonNullable<
             },
           },
         ],
+        workingGroups: [],
```

Comment:
i see you've added the working groups to the data object and now you keep passing empty array in all fixtures, which is unlike what we have been doing for the rest of them. I think it's better if you add some data to it by default as thats the pattern we have been using